python/tox.ini

```diff
@@ -19,31 +19,6 @@ deps =
 passenv =
   PYTEST_ADDOPTS
 commands =
-  py.test -v --reruns 2 --cov --cov-append --cov-report=term-missing sbp tests
+  py.test -v --reruns 2 sbp tests
   {toxinidir}/../test_data/sanity.sh
 sitepackages = False
-
-[coverage:run]
-branch = True
-include = sbp/*
-parallel = True
-
-[coverage:report]
-exclude_lines =
-    # Have to re-enable the standard pragma
-    pragma: no cover
-
-    # Don't complain about missing debug-only code:
-    def __repr__
-    if self\.debug
-
-    # Don't complain if non-runnable code isn't run:
-    if 0:
-    if __name__ == .__main__.:
-ignore_errors = True
-
-[coverage:omit]
-omit =
-    tests/*
-    limbo/*
-    data/*
```

Comment:
This was the source of a lot of headaches. The python tests would randomly crash because of some internal Python code coverage error, it would normally happen with Python 3.10 builds, but not exclusively:

![image](https://github.com/user-attachments/assets/40880e8c-526e-491b-abb5-88194f33a4e8)

There is no need for code coverage here, we are looking to run the test cases and see if they work. If we need code coverage, we can add it later with SonarCloud integration.

Reply:
Maybe someone else from @swift-nav/algint-team can confirm that we don't require coverage right now? That would be my only concern.